src/Symfony/Component/Dotenv/Tests/DotenvTest.php

```diff
@@ -314,4 +314,13 @@ public function testGetVariablesValueFromEnvFirst()
             $this->assertSame('foo2_prod', $values['TEST2']);
         }
     }
+
+    public function testGetVariablesValueFromGetenv()
+    {
+        putenv('Foo=Bar');
+
+        $dotenv = new Dotenv(true);
+        $values = $dotenv->parse('Foo=${Foo}');
+        $this->assertSame('Bar', $values['Foo']);
```

Comment:
please add a `putenv('Foo');` after this line to remove the env var after

Reply:
Should have been in a try-finally.